proto/cosmos/slashing/genesis.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+package cosmos.slashing;
+
+option go_package            = "github.com/cosmos/cosmos-sdk/x/slashing/types";
+option (gogoproto.equal_all) = true;
```

Comment:
Is this needed for all the messages of the file, else i would add it only for corresponding messages